cmd/vic-machine/debug/debug.go

```diff
@@ -192,7 +192,7 @@ func (d *Debug) Run(clic *cli.Context) (err error) {
 	}
 
 	// display the VCH endpoints again for convenience
-	if err = executor.InspectVCH(vch, vchConfig); err != nil {
+	if err = executor.InspectVCH(vch, vchConfig, ""); err != nil {
```

Comment:
I know it might be out of the scope of this issue, but it would be better to show correct connection info to user anytime. 😄